docs/style.md

```diff
@@ -0,0 +1,106 @@
+# Overview
+
+All applicable styles should be colocated with their corresponding JavaScript component using [Aphrodite](https://github.com/Khan/aphrodite).
+
+## Legacy
+
+All legacy styles are processed with [Less](http://lesscss.org/) and can be found in the /less directory. These should still be maintained but all future CSS should be written in JavaScript and kept inside the appropriate component file.
+
+## Example
+
+Here is an example of a React component styled with Aphrodite.
+
+Aphrodite will automatically create a single `<style>` tag in the document `<head>` to put its generated styles in.
+
+```jsx
+const React = require('react')
+const ImmutableComponent = require('./immutableComponent')
+const Aphrodite = require('aphrodite')
+const StyleSheet = Aphrodite.StyleSheet
+const css = Aphrodite.css
```

Comment:
Prefer something like this for the example in one line:

`const {StyleSheet, css} = require('aphrodite')`